homeassistant/components/fjaraskupan/strings.json

```diff
@@ -0,0 +1,20 @@
+{
+  "title": "Fj\u00e4r\u00e5skupan",
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "address": "Device Address"
```

Comment:
Can't we just use the word "Device". I don't think the word "address" adds value since it's a dropdown.

Other thought: why even allow picking up addresses? Why not just discover all devices and set them up automatically, or are we doing some pairing dance?

Reply:
No not really. I suppose we could add all. Not sure how to handle later found devices. I would have to add som reconfigure flow.

Reply:
Couldn't you just do an occasional scan and go from there?

Reply:
Well i need to always scan. It updates data in the announce. So i will see new devices there. So i can just add entities as they are discovered i suppose.

Reply:
I just add all devices found now instead.